standard/expressions.md

```diff
@@ -1059,27 +1070,31 @@ Given two types `T₁` and `T₂`, `T₁` is a ***better conversion target*** th
 >
 > <!-- IncompleteExample: {template:"standalone-lib", name:"OverloadingInGenericClasses", replaceEllipsis:true} -->
 > ```csharp
-> interface I1<T> {...}
-> interface I2<T> {...}
+> interface I1<T> { ... }
+> interface I2<T> { ... }
 >
-> class G1<U>
+> abstract class G1<U>
 > {
->     int F1(U u);               // Overload resulotion for G<int>.F1
->     int F1(int i);             // will pick non-generic
->     void F2(I1<U> a);          // Valid overload
->     void F2(I2<U> a);
+>     abstract public int F1(U u);               // Overload resolution for G<int>.F1
```

Comment:
I'd reorder all of these to "public abstract" - and maybe make the top-level class public as well?
(I do find all the "clutter" of public abstract to be unfortunate. I'm not sure what a better solution would be.)

Reply:
Done